app/src/main/java/ch/epfl/sdp/drone3d/MainActivity.kt

```diff
@@ -24,13 +25,39 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.activity_main_menu)
     }
 
+    private fun refresh() {
+        val loginButton: Button = findViewById(R.id.log_in_button)
+        val logoutButton: Button = findViewById(R.id.log_out_button)
+
+        if (authService.hasActiveSession()) {
+            logoutButton.visibility = View.VISIBLE
+            loginButton.visibility = View.GONE
+        } else {
+            logoutButton.visibility = View.GONE
+            loginButton.visibility = View.VISIBLE
+        }
+    }
+
+    override fun onStart() {
+        super.onStart()
+        refresh()
+    }
+
     /**
      * Go to LoginActivity when log_in_button is clicked
      */
     fun goToLogin(@Suppress("UNUSED_PARAMETER") view: View) {
         open(LoginActivity::class)
     }
 
+    /**
+     * Go to LoginActivity when log_in_button is clicked
+     */
```

Comment:
This is not the good comment for this function

Reply:
Done